packages/pipeline-editor/src/SubmitNotebookButtonExtension.tsx

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Is there a reason git isn't recognizing this as a file name change from `SubmitNotebook.ts` -> `SubmitNotebookButtonExtension.tsx`? They seem to be the same file (with your updates/changes of course). It makes reading git history a bit difficult when files contents are copied to new file instead of renaming them and then editing.

Reply:
Maybe the diff was too big? I think most of the file changed

Reply:
if it wasn't intentional then that's probably the case, oh well